.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: ROS 2 CI
+on:
+  pull_request:
+    paths-ignore: ['**.md', '**.rst', '.gitignore']
+    branches: [main]
+    types: [opened, reopened, synchronize]
+jobs:
+  build_and_test:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: ros-tooling/setup-ros@v0.3
+      with:
+        required-ros-distributions: rolling
+    - uses: ros-tooling/action-ros-ci@v0.2
+      with:
+        package-name: my_package
+        target-ros2-distro: rolling
```

Comment:
Might be useful to also add an artifact saving step to keep the logs?

```yaml
      - name: Upload Logs
        uses: actions/upload-artifact@v3
        with:
          name: colcon_logs
          path: ${{ steps.action_ros_ci.outputs.ros-workspace-directory-name }}/log
        if: always()
```

Reply:
I'm not sure how long they last, but the logs are available without archiving them.
I think this is a good idea for a periodic run, but it doesn't seem necessary to store all the CI logs of all PRs.